index.html

```diff
@@ -0,0 +1,1942 @@
+						</h4>
+					  </article>
+					  <article class="col-md-4">
+						<div class="article-wrap">
+						  <div class="slide-count">
+							<a class="count-link" href="#">40</a>
+						  </div>
+						  <div class="image">
+							<a href="#"><img class="img-fluid" src="images/slideshow-04.jpg"
+								alt="Our Cosplay Favorites from NYCC 2019"></a>
+						  </div>
+						</div>
+						<h4 class="header-3">
+						  <a href="#">Our Cosplay Favorites from NYCC 2019</a>
+						</h4>
+					  </article>
+					</div>
+				  </div>
+				</div>
+			  </div>
+			</div>
+		  </div>
+	
+		  <!-- 
+			HiDPI screeen's stories 6 sub-sections 
+		  -->
+	
+		  <div class="HiDPI featured-stories row">
+			<div class="container-fluid">
+			  <div class="content row col-md-12">
+				<div class="head-story">
+				  <div class="title-HiDPI">
+					<a href="#">U.S.</a>
+				  </div>
+				  <div class="wrap row">
+					<article class="col-md-4">
+					  <div class="article-wrap">
+						<div class="image">
+						  <a href="#"><img class="img-fluid" src="images/HiDPI_01.jpg"
+							  alt="Twitter Asks Space Force Why They Need Camo Uniforms"></a>
+						</div>
+					  </div>
+					  <h4 class="header-3">
+						<a href="#">Twitter Asks Space Force Why They Need Camo Uniforms</a>
+					  </h4>
+					</article>
+					<article class="col-md-4">
+					  <div class="article-wrap">
+						<div class="image">
+						  <a href="#"><img class="img-fluid" src="images/HiDPI_02.jpg"
+							  alt="Top Democrat Blasts Rules Which Could Ease Oversight of Firearms Exports"></a>
+						</div>
+					  </div>
+					  <h4 class="header-3">
+						<a href="#">Top Democrat Blasts Rules Which Could Ease Oversight of Firearms Exports</a>
+					  </h4>
+					</article>
+					<div class="latest-news col-md-4">
+					  <div class="content">
+						<div class="head-story">
+						  <ul class="hidpi-subsection">
+							<li class="d-flex">
+							  <div class="image">
+								<a href="#">
+								  <img class="img-fluid" src="images/HiDPI_03.jpg" alt="opinion journalist">
+								</a>
+							  </div>
+							  <div class="info">
+								<a href="#">Kanye West Faces Criticism for Headlining Event With Anti-LGBTQ Leaders</a>
+							  </div>
+							</li>
+							<li class="d-flex">
+							  <div class="image">
+								<a href="#">
+								  <img class="img-fluid" src="images/HiDPI_04.png" alt="opinion journalist">
+								</a>
+							  </div>
+							  <div class="info">
+								<a href="#">California Convict Kills Pedophile in Prison Attack Says CDCR</a>
+							  </div>
+							</li>
+							<li class="d-flex">
+							  <div class="image">
+								<a href="#">
+								  <img class="img-fluid" src="images/HiDPI_05.jpg" alt="opinion journalist">
+								</a>
+							  </div>
+							  <div class="info">
+								<a href="#">Mega Millions Results, Numbers for 1/17/20: Did Anyone Win the Jackpot?</a>
+							  </div>
```

Comment:
- [ ] The images of the `aside` articles, should be smaller. Check all the breakpoints to make sure they look like the original page.

Reply:
I don't see the purpose here to be smaller, I have customised that way, since it not a need for that , so i felt it easy to go with my way, if you feel I need to change them could you send me the screenshot for the current one & the one with which you want to , I will check on that & do necessary changes if required.